### DISCUSSION TITLE
Not able to change domain

### DISCUSSION THREAD

**Discussion by gl-dev-avinash** [requarks/wiki#4298] [Help / Questions]
I trying to set up Google authentication with wiki js, I did all the setups as suggested in wiki.js docs.

But while redirecting users from `wiki.js` to `google` for authentication, `wiki.js` setting `redirect_uri as https://wiki.yourdomain.com/login/c5494937-e10e-4e2a-b44e-5cd74d8d9e84/callback`

I am not able to change `wiki.yourdomain.com` to the required domain.
FYI: https://www.awesomescreenshot.com/image/11886116?key=510fed0776a64595c20d74ee96304cd5

can anyone please help, its a little urgent?

**Comment by PythonCoderAS** [comment]
What does it say in Site > General > Site Info > Site URL?
